lid.cpp

```diff
@@ -30,6 +35,80 @@ void Lid::validate(std::string filePath)
     return;
 }
 
+void Lid::assembleCodeUpdateImage()
+{
+    info("InbandCodeUpdate: assembleCodeUpdateImage\n");
```

Comment:
do we need this log?

Reply:
Removed it

Reply:
I was debating about the info logs too since in general the direction is not have info logs on good path, but then thought maybe it's ok on inband code update to know where something failed, it'd show how far we got. I'm ok if we leave them for initial debug and we can remove them later, I'm ok if we remove them now too.